crates/vm/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
-name = "essential-constraint-vm"
-version = "0.5.0"
-description = "The Essential constraint checking VM"
+name = "essential-vm"
+version = "0.6.0"
```

Comment:
or 0.1.0?

Reply:
Yeah I wasn't sure which way to go on this 🤔 I think the version `0.6.0` is unchanged from the previous `essential-state-read-vm` version that `develop` is on. Because this is a rename, I think it might be a bit confusing to reset the version entirely? Maybe we should pick the first version that follows the last version of either of the VMs?

Reply:
Yeah that makes sense